app/grants/models.py

```diff
@@ -164,6 +164,27 @@ def percentage_done(self):
         return ((self.amount_received / self.amount_goal) * 100)
 
 
+class Update(SuperModel):
+    """Define the structure of a Grant."""
+
+    title = models.CharField(
+        default='',
+        max_length=255,
+        help_text=_('The title of the Grant.')
+    )
+    description = models.TextField(
+        default='',
+        blank=True,
+        help_text=_('The description of the Grant.')
+    )
+    grant = models.ForeignKey(
```

Comment:
If we want this to show up in activity feeds, we might want to move to `Activity.comment` and just add it there.